endpoints.json

```diff
@@ -73,6 +73,38 @@
           }
         ]
       }
+    },
+    "POST /api/articles/:article_id/comments": {
+  "description": "posts a comment to specified article ",
+  "queries": [],
+  "exampleResponse": {
+    "'comment'":[{
+  
+            "comment_id": 10,
+            "body": "this is cool",
+            "article_id": 1,
+            "author": "lurker",
+            "votes": 0,
+            "created_at": "2020-06-20T07:24:00.000Z"
+          }]
+    }
+  },
+  "PATCH /api/articles/article_id": {
+  "description": "serves an object of article id where votes have been increased",
```

Comment:
You should include the request body example for the post and the patch in this file please.